racetime/models/race.py

```diff
@@ -81,6 +81,12 @@ class Race(models.Model):
         related_name='opened_races',
         null=True,
     )
+    opened_by_bot = models.CharField(
```

Comment:
I do wonder if this should be a `ForeignKey`, but since bots can't be renamed this method is fine, and it does save a query, so maybe just leave it as `CharField`.